assets/style.css

```diff
@@ -88,13 +88,27 @@ h1 {
   border: 1px solid #1a2236;
 }
 
+.Start-collaboration:hover {
+  background-color: #fff;
+  color: #1a2236;
+}
```

Comment:
- [ ] Please as requested by the previous reviewer, you should add the pressed state to meet the design specifications. Use the active sudo class like below and increase the border width. I have used 5px but you should follow the Figma specifications. Note that there is an order in which these sudo classes should be applied.
link, visited, hover, active. make sure active is the last applied style among the 4 sudo states. 👍🏾 

```
.Start-collaboration:active {
   border: 5px solid #1a2236;
}

```